packages/@tinacms/cli/src/cmds/audit/index.ts

```diff
@@ -68,20 +68,40 @@ export const audit = async (ctx: any, next: () => void, options) => {
 
   for (let i = 0; i < collections.length; i++) {
     const collection = collections[i]
-    const returnWarning = await auditCollection({
-      collection,
-      database,
-      rootPath,
-      useDefaultValues: options.useDefaultValues,
-    })
-    const returnError = await auditDocuments({
-      collection,
-      database,
-      rootPath,
-      useDefaultValues: options.useDefaultValues,
-    })
-    warning = warning || returnWarning
-    error = error || returnError
+    try {
+      // Not a huge fan of querying the database from outside of GraphQL
+      // but this allows us to return the list of items unhydrated, so errors
+      // in a single document don't cause the entire list query to fail
+      const docs = await database.query(
+        { collection: collection.name, first: -1, filterChain: [] },
```

Comment:
For using -1 for fetching all pages, will this cause issues for large sites? (I'm not sure if on the backend if we're enforcing any sort of max on the backend, but curious what would happen for sites with thousands of posts)

Reply:
Does audit hit the tina cloud content api?

Reply:
No it doesn't this is probably pretty safe to run on large projects. It only returns the id for each node (and then loops through them 1 by 1).